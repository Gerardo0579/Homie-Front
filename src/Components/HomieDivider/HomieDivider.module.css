.homieDivider{
    margin: 5px 0;
}