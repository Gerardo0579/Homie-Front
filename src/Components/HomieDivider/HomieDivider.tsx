import { Divider } from 'antd'
import { FC } from 'react'
import styles from './HomieDivider.module.css'

export const HomieDivider: FC = () => {
  return <Divider className={styles.homieDivider} />
}
