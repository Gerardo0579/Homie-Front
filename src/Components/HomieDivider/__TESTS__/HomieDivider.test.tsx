import renderer from 'react-test-renderer'
import { HomieDivider } from '../HomieDivider'

it('renders correctly', () => {
  const tree = renderer.create(<HomieDivider />).toJSON()
  expect(tree).toMatchSnapshot()
})
