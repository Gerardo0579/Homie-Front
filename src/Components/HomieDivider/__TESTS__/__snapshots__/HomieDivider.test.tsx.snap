// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-divider ant-divider-horizontal homieDivider"
  role="separator"
/>
`;
